### DISCUSSION TITLE
Spacing presets

### DISCUSSION THREAD

**Discussion by neatlittlemacapps** [six7/figma-tokens#253] [Ideas]
[This magnificent article](https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62) talks about spacing tokens.

Predefined sets of top/bottom/left/right spacings.
Currently you can only obtain this by right clicking a spacing. This makes it not possible to "predefined" a token with a specific spacing combi.
For instance, a "Medium" "Squished Inset" (meaning 16 px left/right and 8px top/bottom).

It would be cool to be able to create predefined spacing tokens for these combi's.

**Comment by six7** [comment]
Definitely, this would be very useful.

This also relates with https://github.com/six7/figma-tokens/issues/110 where we'd have multiple border radii that could be used in combination like this inset idea.
